### ISSUE TITLE
Tests need to be updated

### ISSUE THREAD

**Issue by gj** [learn-co-curriculum/javascript-objects-lab#10]
Student's code, which passes all 5 tests:
```js
var recipes = {}

function updateObjectWithKeyAndValue(object, key, value){
  object[key] = value;
  return object;
}

function destructivelyUpdateObjectWithKeyAndValue(object, key, value){
  object[key] = value;
  return object;
}

function deleteFromObjectByKey(object, key){
  return object;
}

function destructivelyDeleteFromObjectByKey(object, key){
  return object;
}
```
Only the second function, `destructivelyUpdateObjectWithKeyAndValue`, should be passing the tests.

Imagine the assertions need to be updated to better verify object identity/contents, depending on whether the function is destructive or not.

cc: @aturkewi @cernanb

**Comment by gj** [comment]
@aturkewi or @cernanb, please ping @vickdayaram when the tests are updated — he's the student who brought this faulty behavior to my attention.

Thanks!

**Comment by aturkewi** [comment]
Working on this now. Thank you so much for posting the code :) That makes it so much easier to understand and fix 👍

**Comment by aturkewi** [comment]
Fixed in PR #11

**Comment by vickdayaram** [comment]
Sure, no problem, happy to help!
